static/bidder-params/ucfunnel.json

```diff
@@ -0,0 +1,17 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Ucfunnel Adapter Params",
+    "description": "A schema which validates params accepted by the Ucfunnel adapter",
+    "type": "object",
+    "properties": {
+      "adunitid": {
+        "type": "string",
+        "description": "ID for ad unit"
+      },
+      "partnerid": {
+        "type": "string",
+        "description": "ID for partner"
+      }
+    },
+  "required": ["adunitid"]
```

Comment:
Should I set a field as required, if ucfunnel adapter require the field ?

Reply:
Sure. I felt like asking this question because some other adapters in the past have put this or that field as `required` and turns out many of their requests don't come with said parameters and don't get processed.